helm_deploy/values-prod.yaml

```diff
@@ -1,43 +1,26 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+# Environment specific values, override helm_deploy/dps-monitor/values.yaml
+generic-service:
+  replicaCount: 2
 
-replicaCount: 2
+  ingress:
+    host: dps-monitor.prison.service.justice.gov.uk
 
-image:
-  repository: quay.io/hmpps/dps-monitor
-  tag: latest
-  pullPolicy: IfNotPresent
-  port: 3030
+  livenessProbe:
+    httpGet:
+      path: /health
+      port: 3030
+    initialDelaySeconds: 90
+    periodSeconds: 30
+    timeoutSeconds: 20
 
-ingress:
-  enabled: true
-  enable_whitelist: true
-  host: dps-monitor.prison.service.justice.gov.uk
-  cert_secret: dps-monitor-cert
-  path: /
+  readinessProbe:
+    httpGet:
+      path: /health
+      port: 3030
```

Comment:
not sure whether this port should be defaulted to `80` - that is what the `dps-smoketest` is set to